From 8c8f8475ed29f1d160820b38ef89330413746814 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Tue, 17 Aug 2021 16:00:24 +0800
Subject: [PATCH 13/21] fix build
 //tensorflow/lite/examples/label_image:label_image failure

It caused undefined reference neon_tensor_utils failure
...
|tensor_utils.cc:(.text._ZN6tflite12tensor_utils18ReductionSumVectorEPKfPfii+0x0):
undefined reference to
`tflite::tensor_utils::NeonReductionSumVector(float const*,
float*, int, int)'
...

Define Yocto specific cpu `armeabi' for lite label_image
Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 tensorflow/lite/kernels/internal/BUILD | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/tensorflow/lite/kernels/internal/BUILD b/tensorflow/lite/kernels/internal/BUILD
index 2a1f510c131..90710e7751e 100644
--- a/tensorflow/lite/kernels/internal/BUILD
+++ b/tensorflow/lite/kernels/internal/BUILD
@@ -135,6 +135,13 @@ config_setting(
     },
 )
 
+config_setting(
+    name = "armeabi",
+    values = {
+        "cpu": "armeabi",
+    },
+)
+
 config_setting(
     name = "haswell",
     values = {
@@ -287,6 +294,7 @@ selects.config_setting_group(
         ":arm",
         ":arm64-v8a",
         ":armeabi-v7a",
+        ":armeabi",
         ":armhf",
         ":armv7a",
         ":ios_armv7",
-- 
2.25.1

